packages/plugins/plugin-appsflyer/src/AppsflyerPlugin.tsx

```diff
@@ -85,7 +85,7 @@ export class AppsflyerPlugin extends DestinationPlugin {
         },
       };
 
-      if (is_first_launch === 'true') {
+      if (is_first_launch && JSON.parse(is_first_launch) === true) {
```

Comment:
To prevent truthy values but safeguard agains the inconsistent types of AF SDK let's just use the JSON parse value.
```suggestion
      if (JSON.parse(is_first_launch) === true) {
```